db/migrations/20200218211829_home-run-data.js

```diff
@@ -14,6 +14,8 @@ exports.up = function (knex) {
       col.text('access_token');
       col.text('refresh_token');
       col.integer('points');
+      col.integer('permission_level').defaultTo(4);
```

Comment:
Can we move this change into the rc3 file? In a live database we should not alter previous database migration files, and like you have in that file we can use alterTable